### PR TITLE
Translation into Brazilian Portuguese and corrections to the documentation in English.

### PR DESCRIPTION
**Pull request by luizcmarin**
Note the following:
- created folders according to the standard for documentation
- all links checked
- in the asset-manager.md file, on line 186 "[`config/web.php`](../config/web.php)" the file does not exist. I changed it to "[`config/di.php`](../../../config/di.php)" because this file exists.

### CODE REVIEW

File: docs/guides/en/asset-converter.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ After it is done, create `package.json`:
 
 `npm install` brings three packages into `node_modules` directory of our application.
 
-Below we're using bootstrap bundle from "[Asset bundles](asset-bundles.md)" guide:
+Below we're using bootstrap bundle from [Asset bundles](asset-bundles.md) guide:
```
Comment (vjik) [inline docs/guides/en/asset-converter.md:25]: Keep quotes

File: docs/guides/en/asset-manager.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Asset manager
 
 AssetManager resolves asset bundles registered in it and provides a list of files to include into HTML.
-For general usage see "[asset bundles](asset-bundles.md)". In this guide we'll focus on configuring it.
+For general usage see [asset bundles](asset-bundles.md). In this guide we'll focus on configuring it.
```
Comment (vjik) [inline docs/guides/en/asset-manager.md:4]: Keep quotes

**Review by vjik — CHANGES_REQUESTED** [review]
Need fix links to doc files in README

File: README.md [hunk 3]
```diff
@@ -35,46 +35,23 @@ composer require yiisoft/assets --prefer-dist
...
    (such as TypeScript to JavaScript).
 4. Use your favorite method to include files into HTML (out of scope of this package).
 
-## Testing
+## Documentation
 
-### Unit testing
+- [English](docs/guide/en/readme.md)
+- [Portuguese - Brazil](docs/guide/pt-BR/readme.md)
+- More information can be found in the [Internals.](docs/internals.md)
```
Comment (samdark) [inline README.md:49]:
```suggestion
- [Internals](docs/internals.md)
```

**Pull request merged**